app/Http/Controllers/Admin/DigitalPublicationArticleController.php

```diff
@@ -62,12 +72,43 @@ protected function indexData($request)
         ];
     }
 
+    protected function transformIndexItems($items)
+    {
+        // If we're in the browser, don't transform the items
+        if (property_exists($items, 'path')) {
+            return $items;
+        }
+        return $items->toTree();
+    }
+
+    protected function indexItemData($item)
+    {
+        return $item->children ? [
+            'children' => $this->getIndexTableData($item->children),
+        ] : [];
+    }
```

Comment:
This method is already defined as it is in the parent `NestedModuleController`, so no need to redefine it here.